spec/branches/branch_spec.rb

```diff
@@ -0,0 +1,126 @@
+require 'spec_helper'
```

Comment:
May make some shared examples later, enough of these are similar it'd make sense.